src/pages/index.astro

```diff
@@ -123,6 +124,25 @@ posts
       </Container>
     </div>
   </div>
+  <div>
+    <Container
+      class:list={[
+        'flex flex-wrap items-center justify-center gap-8 pt-4 pb-16 md:justify-between',
+      ]}
+    >
+      {
+        sponsors.data.items.map((item) => (
```

Comment:
The acknowledgements section on the donate page also includes institutions that have contributed to Nix in the past (at least we are not sure they do nowadays). Should the landing page continue to show all the past sponsors and supporters forever, only the foundational ones (Utrecht University), or just the current ones?